### ISSUE TITLE
Even though JetStream is disabled, a Persistent Volume Claim gets created

### ISSUE THREAD

**Issue by manifest** [nats-io/k8s#477]
With NATS chart version: 0.15.0 and `nats.jetstream.enabled: false` set in the `values.yaml`, a Persistent Volume Claim gets created.

The current work around to avoid creation of a Persistent Volume Claim is to set `nats.fileStorage.fileStorage.enabled: false` explicitly, but that seems counterintuitive.
```yaml
    nats:
      jetstream:
        enabled: false
        fileStorage:
          enabled: false
```

**Comment by caleblloyd** [comment]
Thanks for the bug report.  This was an oversight when enabling `.Values.nats.jetstream.fileStorage.enabled` by default in 0.15.0.  We should be checking `.Values.nats.jetstream.enabled` also.

**Comment by caleblloyd** [comment]
Should be fixed in `0.15.1`

**Comment by manifest** [comment]
It works. Thank you.
